pipelines/main/platforms/test_macos.yml

```diff
@@ -23,6 +23,11 @@ steps:
           echo "--- Extract build artifacts"
           tar xzf "$${UPLOAD_FILENAME}" "$${JULIA_INSTALL_DIR}/"
 
+          echo "--- Ad-hoc sign for testing"
+          MACHO_FILES=$(find "$${JULIA_INSTALL_DIR}" -type f -perm -0111 | cut -d: -f1)
+          for f in ${MACHO_FILES}; do
+            codesign -s - --option=runtime -vvv --timestamp --deep --force "${f}"
```

Comment:
If possible, let's make use of https://github.com/JuliaLang/julia/blob/master/contrib/codesign.sh here

Reply:
Do we have access to it directly from the julia repo or do I add it to utilities?